CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 All notable changes to this project will be documented in this file.
 
+## [4.4.2] - 2025-07-23
+
+### Security
+
+- Update dependencies
+
+### Fixed
+
+- Don't require JSDoc params & return in TS (#35426 by @diondiondion)
+- Disable `useDrag` hook when main menu is not openable (#35414 by @diondiondion)
```

Comment:
Maybe bundle this with #35390 since they are trying to fix the same issue afaict?